node/parachain/src/command.rs

```diff
@@ -152,7 +161,7 @@ pub fn run() -> Result<()> {
 					task_manager,
 					import_queue,
 					..
-				} = crate::service::new_partial(&config)?;
+				} = crate::service::new_partial(&config, account)?;
```

Comment:
We don't need an author address when purging the chain. This match branch is `CheckBlock`. The `PurgeChain` branch does not use the author address.

Reply:
Good catch! There's a few branches where it's not used in `new_partial` to get the return values so in those cases I pass in H160::default(). It could definitely be written better and I'm looking into that now.

Reply:
I think this is actually a bug. The import queue returned was instantiated with an inherent data provider with account id H160::default. So the returned import queue is invalid. All of the paths that return `import_queue` need to be removed from the `no_account` path. Doing it now.

Reply:
We need to make sure the inherent data provider passed to `cumulus_import_queue` doesn't call the `set-author` inherent because that's not intended, but it is what is happening now. So the `build_inherent_data_provider` function needs to change the account param to an `Option<AccountId>` so it's None when we only need inherent data providers for cumulus and don't want to set the block author as well.